oak/server/storage/fixed_nonce_generator.h

```diff
@@ -17,6 +17,8 @@
 #ifndef OAK_SERVER_STORAGE_FIXED_NONCE_GENERATOR_H_
 #define OAK_SERVER_STORAGE_FIXED_NONCE_GENERATOR_H_
 
+#include <openssl/sha.h>
```

Comment:
Does there need to be `BUILD` change for the new direct dependency here?

Reply:
Bazel didn't think so.  It actually builds without the include.  🤷‍♂

Reply:
Yeah, but if whatever transitive target happens to change to no-longer include OpenSSL, won't this target mysteriously start to fail?

Reply:
We probably need to add boringssl to the WORKSPACE file.  Will follow up with another PR.

Reply:
I think it's already pulled in by Asylo, otherwise it would not even compile. IIUC only the BUILD dependency is missing here.